src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md

```diff
@@ -57,19 +55,16 @@ The product page has different sections:
 2. **Team contacts:** Product Owner and Technical Lead(s). The contacts need to have an IDIR, if you need to change them, editing the email field will autopopulate their new information.
 
 3. **Quotas:**   If you need more resources for CPU, RAM or storage in any of the four namespaces, this is the place to make changes to them. This is also the section where we will be focusing on this guide.
-
  * Production `prod`
  * Test `test`
  * Tools `tools`
  * Development `dev`
-
+ 
 4. **Common components** This is an optional field where you can choose different components through a list of them.
 
 ### Step 3: Prepare the request 
 
-In the **quotas** section of the page adjust your new quota request for each of the 4 namespaces `prod`, `test`, `tools`, `dev` if needed. 
-
-Use the following guidelines, depending on the resource type:
+In the **quotas** section of the page adjust your new quota request for each of the 4 namespaces `prod`, `test`, `tools`, `dev` if needed. Use the following guidelines, depending on the resource type:
 
   **CPU or RAM quota increase**
 
```

Comment:
tables in this section don't display a scroll bar to indicate there are more columns. this would be a good improvement to the site

Reply:
Displaying different in browsers I believe